modules/ocs2/src/main/scala/gem/ocs2/SmartGcalImporter.scala

```diff
@@ -175,35 +174,40 @@ object SmartGcalImporter extends TaskApp with DoobieClient {
                     parser:         KeyParser[K],
                     unindexer:      ConnectionIO[Int],
                     writer:         SmartGcalWriter[K],
-                    indexer:        ConnectionIO[Int]): Task[Unit] = {
+                    indexer:        ConnectionIO[Int]): IO[Unit] = {
 
-    def lines(l: GcalLampType): Process[Task, SmartGcalLine[K]] =
-      scalaz.stream.io
-          .linesR(new File(dir, fileName(instFilePrefix, l)).getPath)
+    def lines(l: GcalLampType): Stream[IO, SmartGcalLine[K]] =
+      io.file.readAll[IO](new File(dir, fileName(instFilePrefix, l)).toPath, 4096)
+          .through(text.utf8Decode)
+          .through(text.lines)
           .map(_.split(',').map(_.trim).toList)
           .map(parseLine(_, l, parser))
```

Comment:
For some reason an empty line is being inserted, which blows up the parser.  The parser doesn't try to handle anything other than exactly what is produced by the smart-gcal exporter.  An ad-hoc solution would be to filter them, I suppose, but I do wonder where the blank line comes from.  I assume it has to do with `text.lines`.

```
...
List(R3000, K_SHORT, LONGSLIT_8, 1, NIR balance, IR, Ar arc, Closed, 80000, 1, NIGHT)
List(R3000, J_LOW, LONGSLIT_4, 1, NIR balance, IR, Ar arc, Closed, 30000, 1, NIGHT)
List(R1200HK, HK, LONGSLIT_3, 1, NIR balance, IR, Ar arc, Closed, 90000, 1, NIGHT)
List(R3000, J, LONGSLIT_3, 1, NIR balance, IR, Ar arc, Closed, 30000, 1, NIGHT)
List()
[error] (run-main-2) scala.MatchError: List() (of class scala.collection.immutable.$colon$colon)
scala.MatchError: List() (of class scala.collection.immutable.$colon$colon)
	at gem.ocs2.SmartGcalImporter$.parseF2(SmartGcalImporter.scala:63)
...
```